### PR TITLE
Create Issue-Templates.md

### PR DESCRIPTION
**Pull request by Mstronach**
Issue templates for bug, enhancement and question now exist within the issue-templates.md

### CODE REVIEW

File: Issue-Templates.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Issue Templates
+
+## 🌱 Describe your Feature Request
+- A clear and concise description of what the problem is.
+- CoreML / iOS / TuriCreate version you are using?
```
Comment (TobyRoseman) [inline Issue-Templates.md]: `CoreML / iOS / TuriCreate` -> `TuriCreate`.

File: Issue-Templates.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+If applicable, please paste the error trace.
+
+## To Reproduce
+- If a python script can reproduce the error, please paste the code snippet
+```
+# Paste code snippet here
+```
+- If applicable, please attach turi / source model
+    - If model cannot be shared publicly, please attach it via filing a bug report at https://developer.apple.com/bug-reporting/ 
+- If model conversion succeeds, however, there is numerical mismatch between the original and the coreml model, please paste python script used for comparison (turi code etc.)
```
Comment (TobyRoseman) [inline Issue-Templates.md]: Let's just remove these three lines.

File: Issue-Templates.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+## System environment (please complete the following information):
+ - coremltools version  (e.g., 3.0b5):
```
Comment (TobyRoseman) [inline Issue-Templates.md]: `- coremltools version  (e.g., 3.0b5):` -> `- turicreate version  (e.g., 5.7.1):`

File: Issue-Templates.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+ - OS (e.g., MacOS, Linux):
+ - macOS version (if applicable):
+ - XCode version (if applicable):
+ - How you install python (anaconda, virtualenv, system):
+ - python version (e.g. 3.7):
+ - Turi version (e.g. 5.7.1):
+ - any other relevant information:
+     - e.g. keras version if using keras conversion etc.
```
Comment (TobyRoseman) [inline Issue-Templates.md]: This line doesn't make sense for us.

**Comment by Mstronach** [comment]
Closing this as create issue templates will be created via github settings. Changes to templates based on comments have been made and will be used for each issue type.

**Pull request closed without merging**